### ISSUE TITLE
After deactivating an inertial sensor in 'mtbSensorAct' in the main user interface, it is still added to the sensors list in 'data'

### ISSUE THREAD

**Issue by nunoguedelha** [nunoguedelha/jointOffsetCalibInertial#2]
This issue occured while testing on the real robot, using the Matlab app script commit [2ce8cec](https://github.com/nunoguedelha/jointOffsetCalibInertial/commit/2ce8cec9231fca0bc4e308e600c72dd2bb0c7663).
Although in `sensorSelfCalibratorInit`, we set the following configuration options (10b10_acc off):
```
acquireSensorsTestData  = true;
mtbSensorAct.left_arm = [10:13 8:9 7];
mtbSensorAct.right_arm = [10:13 8:9 7];
mtbSensorAct.left_leg = [1:9 11:13];
mtbSensorAct.right_leg = 1:13;
mtbSensorAct.torso = 7:10;
mtbSensorAct.head = 1;
```
after the acquired data is loaded, we still get in the `data.bc` structure:
```
>> data.bc.labels
ans =
  1×27 cell array
  Columns 1 through 6
    '10b1_acc'    '10b2_acc'    '10b3_acc'    '10b4_acc'    '10b5_acc'    '10b6_acc'
  Columns 7 through 12
    '10b7_acc'    '10b8_acc'    '10b9_acc'    '10b10_acc'    '10b11_acc'    '10b12_acc'
  Columns 13 through 18
    '11b1_acc'    '11b2_acc'    '11b3_acc'    '11b4_acc'    '11b5_acc'    '11b6_acc'
  Columns 19 through 24
    '11b7_acc'    '11b8_acc'    '11b9_acc'    '11b10_acc'    '11b11_acc'    '11b12_acc'
  Columns 25 through 27
    '11b13_acc'    'left_leg_state'    'right_leg_state'
```

**Comment by nunoguedelha** [comment]
Activated inertial MTB sensors should be : [1 2 3 4 5 6 7 8 9 11 12 13]
Instead we have : [1 2 3 4 5 6 7 8 9 10 11 12].
Cause: `mtbSensorAct` is no more a list of booleans, but is instead a vector of sensor indexes.
Root cause: `mtbSensorAct` was changed in `.../src/conf/sensorSelfCalibratorInit.m` but the change was not propagated here.
Checked that no other occurrences of `mtbSensorAct` handling are impacted by this issue.

**Comment by nunoguedelha** [comment]
Fixed by commit 5122e6f9fc0f3f91aa12ee7e2c8734a5c98136d9
